docs/guide/tutorials.rst

```diff
@@ -0,0 +1,431 @@
+
+	.. image:: ../images/hplc_uv_standards_exports.png
+
+	Based in the data you wish to export, select the desired "Sample types" from the left pane
+	and select the "Metadata" from the right pane then click on of the buttons below to create
+	the report with the selected items in the csv format. More details on exporting the results can be found 
+	in :ref:`Export report`.
+
+
+
+
+Targeted quantitation with LC-MS/MS 5500 QTRAP RapidRIP
+-------------------------------------------------------
+
+
+
+Targeted flux analysis with LC-MS/MS 5500 QTRAP
+-----------------------------------------------
+
+
+
+Targeted flux analysis with GC-MS full-scan Agilent
+---------------------------------------------------
+
+
+
+Targeted flux analysis with GC-MS SIM Agilent
+---------------------------------------------
+
+
+
+Non-targeted FIA-MS analysis with Thermo Orbitrap
+-------------------------------------------------
+
+This tutorial walks you through the workflow for analyzing targeted HPLC data
+starting from input file generation, to processing the data in SmartPeak, 
+to reviewing the data in SmartPeak, to reporting the results for later use.
+
+Objectives
+~~~~~~~~~~
+
+#. Obtaining the SOP for the workflow.
+#. Choosing a data set for demonstrating the workflow.
+#. Creating an optimized SmartPeak input templates for running the workflow.
+
+The Workflows include
+~~~~~~~~~~~~~~~~~~~~~
+
+#. Calculating the calibration curves using Standards
+#. Processing Unknowns
+
+Steps
+~~~~~
+
+The tutorial includes the following steps :
+
+#. Setting up the input files
+
+The data set used can be found here 
+`FIAMS FullScan Unknowns <https://github.com/AutoFlowResearch/SmartPeak/tree/develop/src/examples/data/FIAMS_FullScan_Unknowns>`_.
+
+#. Defining the workflow in SmartPeak
+
+For FIAMS FullScan Unknowns analysis, the following steps are saved 
+into the ``workflow.csv`` file. Alternatively, steps can be replaced, 
+added or deleted direclty from SmartPeakGUI within the "workflow" tap in the right pane. 
+A detailed explanation of each command step
+can be found in :ref:`Workflow Commands`.
+
+	* LOAD_RAW_DATA
+	* EXTRACT_SPECTRA_WINDOWS
+	* MERGE_SPECTRA
+	* PICK_MS1_FEATURES
+	* SEARCH_ACCURATE_MASS
+	* STORE_ANNOTATIONS
+	* STORE_FEATURES
+	* ESTIMATE_FEATURE_BACKGROUND_INTERFERENCES
+	* STORE_FEATURE_BACKGROUND_ESTIMATIONS
+	* FILTER_FEATURES_BACKGROUND_INTERFERENCES
+	* MERGE_FEATURES
+	* MERGE_INJECTIONS
+	* STORE_FEATURES_SAMPLE_GROUP
+
+	The Spectra for the two injection samples can be inspected after all workflow steps had been run, to do so please
+	click on view and then "Spectra". From the Injections tab check "Plot/Unplot All" select all injection samples and 
+	plot the mass to charge ratio relative to their respective intensities as shown below:
+
+	.. image:: ../images/fiams_fullscan_unknowns_spectra.png
+
+	#. Reporting the results
+
+	To export the results, select "Report" from the "Actions" which will show the 
```

Comment:
This is a good opportunity to discuss the export options for the `Sample_Group` reports.  Also, please discuss the different Feature files (i.e., from the raw data processor and from the sample group processor) that can be stored.  I would also recommend noting that these can be uploaded and parsed directly using python using the pyOpenMS package.